code/src/components/GameScreen.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux';
+import game, { generateGame } from 'reducers/game';
+
+import GameImages from './GameImages';
+
+const GameScreen = () => {
+  const step = useSelector((store) => store.game.currentStep);
+  const history = useSelector((store) => store.game.history);
+
+  const dispatch = useDispatch();
+
+  const onNextStep = (action) => dispatch(generateGame(action));
+  const onPreviousStep = () => dispatch(game.actions.setPreviousStep());
+
+  const directions = (action) => {
+    const moveString = `Move ${action.direction}`
+
+    if (action.direction === 'North') {
+      return moveString + '↑'
+    } else if (action.direction === 'South') {
+      return moveString + '↓'
+    } else if (action.direction === 'West') {
+      return moveString + '←'
+    } else {
+      return moveString + '→'
+    }
+  };
+
+  return (
+    <section className='game-container'>
+      {step.actions.length === 0 && <i className='nes-icon trophy is-large' />}
```

Comment:
This works, if you want to make it more readable, and since it's used in multiple places, consider creating a boolean like:
```
const isOver = step.actions.length === 0
``` 
...which will make it even easier to read.
```
{isOver && <i className='nes-icon trophy is-large' />}